WcaOnRails/app/javascript/edit-schedule/EditVenue/index.jsx

```diff
@@ -110,6 +117,24 @@ const NameInput = ({name, actionHandler}) => (
   </Row>
 );
 
+const CountryInput = ({value, onChange}) => (
+  <Row>
+    <Col xs={3}>
+      <span className="venue-form-label control-label">Country:</span>
+    </Col>
+    <Col xs={9}>
+      <select
+        className="form-control"
+        value={value}
+        onChange={onChange}
+      >
+        <option key="" value="" />
```

Comment:
I believe we don't need that =)

Reply:
Ah, I added it before doing a backfill, and before updating the code to auto-select the right country. You're right, we don't need this anymore!